chapter2/tests/t13/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1.2 
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14-slim AS builder
```

Comment:
Use the `ARG` instruction to transfer the version of the source image

Reply:
> Use the `ARG` instruction to transfer the version of the source image

@udaltsovra 
Done